openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -267,12 +267,29 @@ def write_project_to_op(project: dict, dbcon: AvalonMongoDB) -> UpdateOne:
         # Update Zou
         gazu.project.update_project(project)
 
+    all_kitsu_projects_names = [
+        project['name'] for project in gazu.project.all_projects()
+    ]
+    all_kitsu_open_projects_names = [
+        project['name'] for project in gazu.project.all_open_projects()
+    ]
+    all_kitsu_closed_projects_names = [
+        project for project in all_kitsu_projects_names
+        if project not in all_kitsu_open_projects_names
+    ]
```

Comment:
```suggestion
    all_kitsu_projects_names = {
        project['name'] for project in gazu.project.all_projects()
    }
    all_kitsu_open_projects_names = {
        project['name'] for project in gazu.project.all_open_projects()
    }
    all_kitsu_closed_projects_names = (
        all_kitsu_projects_names  - all_kitsu_open_projects_names 
    )
```

I assume the lists of projects can grow quite large over time - especially if it includes the non-open ones. If we end up doing lookups like this I'd say we should go for `set` type.